CHANGELOG.md

```diff
@@ -3,10 +3,14 @@
 ## [Unreleased]
 ### Added
 * Self-Assign button for Flaws
-* Time to Public Date field
+* Disable form on references and acknowledgments save actions
 
 ### Fixed
 * The session is now shared across tabs
+* References and acknowledgments disappear after save actions
+* References and acknowledgments are not refreshed after save actions
+* Time to Public Date field
```

Comment:
Can you change to `* Provide time to Public Date field` and move to added section? I think I messed this up on my changelog edits 😅